docs/src/props-table.js

```diff
@@ -107,10 +107,12 @@ function PassthroughPropsRow({elementName, isPolymorphic, passthroughPropsLink})
 }
 
 function AsRow({defaultElementType}) {
+  // Element is a component if the first letter is uppercase (e.g. Button)
+  const isComponent = defaultElementType[0].toUpperCase() === defaultElementType[0]
   return (
     <Row
       name="as"
-      defaultValue={`"${defaultElementType}"`}
+      defaultValue={isComponent ? defaultElementType : `"${defaultElementType}"`}
```

Comment:
This prevents component names from being displayed as strings (`"Overlay"` vs `Overlay`)